src/evlib/processing/reconstruction/e2vid/e2vid.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import os
+import logging
+import sys
+from typing import Tuple
+
+import numpy as np
+import urllib.request
+
+from .utils.loading_utils import load_model, get_device
+from .image_reconstructor import ImageReconstructor
+from .options.inference_options import set_inference_options
+from ....representation import VoxelGrid  # TODO: import correctly from evlib
```

Comment:
I was not sure here. Can also do 

`from evlib.representation import VoxelGrid`

?